<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentDialog</class>
 <widget class="QDialog" name="AppointmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预约信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_patient">
       <property name="text">
        <string>病人ID*:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="patientIdSpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_doctor">
       <property name="text">
        <string>医生ID*:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="doctorIdSpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>预约日期*:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="appointmentDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>预约时间*:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="appointmentTimeEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string>状态:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="statusCombo"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_reason">
       <property name="text">
        <string>原因:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="reasonEdit">
       <property name="maximumHeight">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_notes">
       <property name="text">
        <string>备注:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="notesEdit">
       <property name="maximumHeight">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppointmentDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppointmentDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
